themebuilder-scss/.eslintrc-themebuilder

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "parserOptions": {
+        "project": "./tsconfig.json",
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    },
+    "extends": [
+        "devextreme/typescript"
```

Comment:
Let's try to add also:
```
"eslint:recommended"
"plugin:@typescript-eslint/eslint-recommended"
```
?

See: 
https://www.npmjs.com/package/@typescript-eslint/eslint-plugin#recommended-configs